app/experimenter/static/rapid/components/experiments/ExperimentDetails.tsx

```diff
@@ -48,13 +48,20 @@ const ExperimentDetails: React.FC = () => {
     // No-op
   };
 
+  let bugzilla_url;
+  if (data.bugzilla_url) {
+    bugzilla_url = (
+      <div className="my-2">
+        Bugzilla ticket can be found <a href={data.bugzilla_url}>here</a>.
```

Comment:
Not sure if you want a `target="_blank" rel="noopener noreferrer"` bit on this to spawn the bug in a new window to avoid losing the current Experimenter window. I'm mildly indifferent, but thought it was worth calling out in case anybody has strong feelings one way or the other (or if Experimenter has prior art on these links).

Reply:
oooh yes